### PR TITLE
[MTM-50741] Updated screenshots for Administration > Managing data (en + de)

### PR DESCRIPTION
**Pull request by pawel-rynarzewski-sag**
Peek 2023-01-27 14-27

Peek 2023-01-27 14-28

### CODE REVIEW

File: content/benutzerhandbuch/administration-de-bundle/data-retention.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Datenhaltungsregeln werden üblicherweise während der Nacht ausgeführt. Wenn S
 
 Klicken Sie auf **Datenhaltungsregeln** im Menü **Verwaltung**, um eine Liste aller Datenhaltungsregeln in Ihrem Konto anzuzeigen.
 
-<img src="/images/benutzerhandbuch/Administration/admin-retention-rules.png" alt="Retention rules">
+<img src="/images/benutzerhandbuch/Administration/admin-retention-rules.png" alt="Datenhaltungsregeln">
```
Comment (BeateRixen) [inline content/benutzerhandbuch/administration-de-bundle/data-retention.md:17]: @pawel-rynarzewski-sag Just FYI. These image links will be overwritten with the next translation cycle as we create the German files based on the English ones, just replacing the text but not the functional elements. We only replace the folder name for the images with a script. This is way all these alternative texts are in English.

Comment (pawel-rynarzewski-sag) [reply content/benutzerhandbuch/administration-de-bundle/data-retention.md:17]: ok, I see :+1:

**Comment by jkot-softwareag** [comment]
My suggestion would be to improve this image: https://github.com/SoftwareAG/c8y-docs/pull/809/files#diff-2adbb1b8fcae55464bde050e20f107d7663cb0ae817e9043ec4b10cfeb457842
by adding few files with more meaningful names and different types. In terms of "business logic" I like the previous screenshot more.

**Comment by BeateRixen** [comment]
@pawel-rynarzewski-sag Should I merge this PR?

**Pull request merged**